pkg/standalone/client.go

```diff
@@ -0,0 +1,32 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package standalone
+
+type DaprProcess interface {
+	List() ([]ListOutput, error)
+}
+
+type daprProcess struct {
+}
+
+type Client interface {
+	DaprProcess
```

Comment:
Do we need separate DaprProcess from Client ? Can we just have `Client` including List() ?

Reply:
We can, but the tests that are using list, it needs to be mocked ...If the client also includes list, then the mock client for the test needs to implement all methods to confirm to the interface.

Reply:
I presume that you split to two parts because DaprProcess maintains Process of dapr runtime and Clients uses http client to call Dapr API endpoint. then it makes sense, but I do not think we need to embed DaprProcess to Client. 

The actual struct implementing Client interface can have DaprProcess instead of embedding it to Client interface. What do you think?

Reply:
Yes that works too. Will change.